src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design/project.json

```diff
@@ -1,6 +1,6 @@
 {
-  "version": "1.0.0-*",
-  "description": "Razor precompilation",
+  "version": "1.1.0-*",
+  "description": "Razor view compilation",
```

Comment:
> detect Razor syntax errors at compilation time

The samples in this repo generally have it the precompilation set up as postpublish not post compile. Not that it wouldn't work in postcompile.